pkg/cmd/flags/policy.go

```diff
@@ -0,0 +1,219 @@
+package flags
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/aquasecurity/tracee/pkg/errfmt"
+	"github.com/aquasecurity/tracee/pkg/events"
+	"github.com/aquasecurity/tracee/pkg/filterscope"
+	"github.com/aquasecurity/tracee/types/policy"
+)
+
+// receive array of policies
+func PrepareFilterScopesForPolicies(policies []policy.Policy) (*filterscope.FilterScopes, error) {
+	parsedMap := map[int][]*filterFlag{}
+
+	// max 64?
+	for i, p := range policies {
+		err := validatePolicy(p)
+		if err != nil {
+			return nil, err
+		}
+
+		filterFlags := make([]*filterFlag, 0)
+
+		if p.Scope == nil {
+			return nil, errfmt.Errorf("scope cannot be empty")
+		}
+		// scope
+		for _, s := range p.Scope {
+			if s == "global" {
+				continue
+			}
+
+			operatorIdx := strings.IndexAny(s, "=!<>")
+
+			err := validateScope(s[:operatorIdx])
```

Comment:
I think that the `!container` causes the `slice bounds out of range [:-1]` here. We need to consider this prefix operator. Take a look at https://github.com/aquasecurity/tracee/blob/a04d8e7d7c0fc7cabdbc9bb0d6e55a410e3f7aac/pkg/cmd/flags/filter.go#L141-L159